pkg/serverless/daemon/interval_test.go

```diff
@@ -23,22 +23,28 @@ func TestAutoSelectStrategy(t *testing.T) {
 
 	now := time.Now()
 
+	// prefilling lastInvocations with 17 timestamps since we need 20 to change flush strategies
+	for i := 0; i < 17; i++ {
+		d.StoreInvocationTime(now.Add(time.Second * time.Duration(i)))
+	}
+
+	fmt.Print(len(d.lastInvocations))
 	// when not enough data, the flush at the end strategy should be selected
 	// -----
 
 	assert.Equal((&flush.AtTheEnd{}).String(), d.AutoSelectStrategy().String(), "not the good strategy has been selected") // default strategy
 
-	assert.True(d.StoreInvocationTime(now.Add(-time.Second * 140)))
+	assert.True(d.StoreInvocationTime(now.Add(time.Second * 18)))
 	assert.Equal((&flush.AtTheEnd{}).String(), d.AutoSelectStrategy().String(), "not the good strategy has been selected")
-	assert.True(d.StoreInvocationTime(now.Add(-time.Second * 70)))
+	assert.True(d.StoreInvocationTime(now.Add(time.Second * 19)))
 	assert.Equal((&flush.AtTheEnd{}).String(), d.AutoSelectStrategy().String(), "not the good strategy has been selected")
 
 	// add a third invocation, after this, we have enough data to decide to switch
```

Comment:
Looks like this comment is out of date now, probably best to just remove it and let the code speak for itself.